src/main/java/namor/template/main/App.java

```diff
@@ -1,9 +1,64 @@
 package namor.template.main;
```

Comment:
after the template gotes package main. Why what is there so "main" about stack ?